core/broker/broker.go

```diff
@@ -135,6 +136,10 @@ func (b *broker) Init(c *component.Component) error {
 	b.Component.SetStatus(component.StatusHealthy)
 	if b.Component.Monitor != nil {
 		b.monitorStream = b.Component.Monitor.NewBrokerStreams(b.Identity.Id, b.AccessToken)
+
+		if len(viper.GetStringMapString("monitor-servers")) > 0 {
```

Comment:
Let's leave it like this until we refactor the configuration system (do you agree, @johanstokking?)